common/component_interface_utils/CMakeLists.txt

```diff
@@ -3,5 +3,5 @@ project(component_interface_utils)
 
 find_package(autoware_cmake REQUIRED)
 autoware_package()
-
+ament_export_dependencies(tier4_system_msgs)
```

Comment:
I think it's automatically exported by `ament_auto_package`.
https://github.com/ament/ament_cmake/blob/799183ab9bcfd9b66df0de9b644abaf8c9b78e84/ament_cmake_auto/cmake/ament_auto_package.cmake#L58

```suggestion
```

Reply:
I thought so too, but actually the build fails without this.

Reply:
Hmm, I see. :thinking: 
Okay, thanks!